resources/users/auth/auth.controller.js

```diff
@@ -36,22 +36,12 @@ module.exports = function (Model) {
           password: hash,
         });
 
-        const payload = {
-          id: doc.id,
-          username: doc.username,
-        };
```

Comment:
Good job @twumm, well done. I thought you suggested we shouldn't include username in the payload, I see you implemented that in the login.

Reply:
Hey @tolaked this code(with a pinkish highlight) indicates what has been removed.
You need to sleep sha!
Watch the code highlighted in the light shade of green.